lib/theme/constant.dart

```diff
@@ -0,0 +1,22 @@
+class Metrics {
+  static const defaultBorderRadius = 12.0;
+  static const defaultHorizontalPadding = 24.0;
+
+  static const pageIndicatorSize = 8.0;
+  static const pageIndicatorAnimationDuration = 200;
+
+  static const profileSmallRadius = 18.0;
+
+  static const spacingXXLarge = 79.0;
+  static const spacingXLarge = 60.0;
+  static const spacingLarge = 54.0;
+  static const spacingMedium = 26.0;
+  static const spacingDefault = 20.0;
+  static const spacingSmall = 16.0;
+  static const spacingXXSmall = 4.0;
+
+  static const fontLarge = 34.0;
+  static const fontMedium = 28.0;
+  static const fontNormal = 22.0;
+  static const fontXSmall = 13.0;
```

Comment:
It seems we don't use any dimens in `Metric`, Maybe just leave todo and add it later 🙏 
```suggestion
// TODO: Add the dimension here
```